content/en/config.yaml

```
@@ -30,8 +30,8 @@ params:
       subtitle: Modify and create your own OpenMS workflows in KNIME.
       image: knime-edit.png
       buttonText: Get Started
-      buttonLink: "https://openms.readthedocs.io/en/latest/run-workflows-with-openms-tools/overview.html"
-    - title: Execute python bindings 🐍
+      buttonLink: "https://openms.readthedocs.io/en/latest/run-workflows-with-openms-tools/openms-in-knime.html"
+    - title: Execute python bindings
```

bindings are not executed
```suggestion
    - title: Use OpenMS in python
```